README.md

```diff
@@ -9,9 +9,28 @@ Also included:
   + APIs for packet data over radio [README_data](README_data.md)
   + An STM32 embedded version of FreeDV 1600/700D/700E for the [SM1000](stm32/README.md)
 
-## Older code
+## Old Code and Deprecated Features
 
-In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev
+In July 2023 this repo was refactored, older code can be found in https://github.com/drowe67/codec2-dev. 
+
+We are currently conducting a major re-development of Codec 2, new speech coding, modems, and FreeDV modes are under active development. As such, we have limited resources available for mantinence or development on deprecated features unless there is a major bug. We'd rather put our efforts into new features! Deprecated modes include:
+1. FreeDV 1600, 700C, 700D, 700E, 2020
```

Comment:
Generally I would not deprecate modes until there's something available to replace them. It would probably be more accurate just to say that we're only accepting bug fixes for the stuff we have now. If we are deprecating stuff, I'd recommend deprecating the following:

* 700C
* 2020/2020B
* Codec2 modes *besides* 1600/3200 (for M17) and whatever else is used by 700D/E/1600.
* LPCNet

Reply:
I agree with:
> Generally I would not deprecate modes until there's something available to 
> replace them. 

But don't feel this captures our PLT decision:
> It would probably be more accurate just to say that we're only accepting bug 
> fixes for the stuff we have now.
(unless it is a major bug - which I have covered in the text above.)

Just because we have it now, is not a good reason for our resources to be diverted away from important R&D to work on it.

The key point here is to avoid diverting important resources to non-critical bug fixes and maintenance to code that is hopefully (if we meet our performance goals) going to be deprecated soon.

Reply:
@tmiw - I've had another go at the wording in https://github.com/drowe67/codec2/pull/26/commits/aa1d6c0b977a178de55c18a4b026d3c057f7d868 - see what you think